aspnet/data/entity-framework-6.rst

```diff
@@ -1,110 +1,99 @@
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Since your EF6 context doesn't have a parameterless constructor, your EF6 project has to provide an implementation of `IDbContextFactory <https://msdn.microsoft.com/library/hh506876>`__. The EF6 command-line tools will find and use that implementation so they can instantiate the context. Here's an example.
+
+.. literalinclude::  entity-framework-6/sample/EF6/SchoolContextFactory.cs
+  :language: c#
+  :start-after: snippet_IDbContextFactory
+  :end-before:  #endregion
+  :dedent: 4
 
-This is the default and so no explicit configuration is needed. The above ``CodeConfig`` class can be used to explicitly set the provider services and the appropriate connection string should be passed to the ``DbContext`` constructor as shown `above <#setup-connection-strings-and-dependency-injection>`_.
+In this sample code, the ``IDbContextFactory`` implementation passes in a hard-coded connection string. This is the connection string that the command-line tools will use. You'll want to implement a strategy to ensure that the class library uses the same connection string that the calling application uses. For example, you could get the value from an environment variable in both projects.
+   
+Set up dependency injection in the ASP.NET Core project
+-------------------------------------------------------
+
+In the Core project's *Startup.cs* file, set up the EF6 context for dependency injection (DI) in ``ConfigureServices``. EF context objects should be scoped for a per-request lifetime.
+
+.. literalinclude::  entity-framework-6/sample/MVCCore/Startup.cs
+  :language: c#
+  :start-after: snippet_ConfigureServices
+  :end-before:  #endregion
+  :dedent: 8
+
+You can then get an instance of the context in your controllers by using DI. The code is similar to what you'd write for an EF Core context:
+
+.. literalinclude::  entity-framework-6/sample/MVCCore/Controllers/StudentsController.cs
+  :language: c#
+  :start-after: snippet_ContextInController
+  :end-before:  #endregion
+  :dedent: 4
+
+Sample application
+------------------
+
+For a working sample application, see the `sample Visual Studio solution <https://github.com/aspnet/Docs/tree/master/aspnet/data/entity-framework-6/sample/>`__ that accompanies this article.
+
+This sample can be created from scratch by the following steps in Visual Studio:
+
+* Create a solution.
+* **Add New Project > Web > ASP.NET Core Web Application (.NET Framework)**
+* **Add New Project > Windows > Class Library**
+* In **Package Manager Console** (PMC) for both projects, run the command ``install-package entityframework``.
+* In the class library project, create data model classes and a context class, and an implementation of ``IDbContextFactory``.
+* In PMC for the class library project, run the commands ``enable-migrations`` and ``add-migration Initial``. If you have set the ASP.NET Core project as the startup project, add ``-StartupProjectName EF6`` to these commands.
```

Comment:
Nit: To match the casing of the actual commands, you should probably use
* `Install-Package EnityFramework`
* `Enable-Migrations`
* `Add-Migration Initial`